Informatics I/010 Modularization and Git/Task 1 International Banking/Perytron Solution/currency_converter.py

```diff
@@ -0,0 +1,29 @@
+from typing import Union
+from exchange_rates import EXCHANGE_RATES
+
+def convert(amount: Union[int, float], from_currency: str, to_currency: str) -> float:
+
+    # Check that correct type is provided
+    if False in [isinstance(from_currency, str), isinstance(to_currency, str), (isinstance(amount, float) or isinstance(amount, int))]: raise Warning("You had one (...) job. Provide the right type you (...) (...)!")
```

Comment:
fancy solution! i like it. But would layout diffrently, even if one-liners are neet, max suggested line length for python is 79, if longer try formatting it into multiply lines for readabilty.
[python best line length](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjg34qWtNv7AhW57rsIHef9ChsQFnoECAoQAw&url=https%3A%2F%2Frealpython.com%2Fpython-pep8%2F%23%3A~%3Atext%3Dto%2520one%2520another.-%2CMaximum%2520Line%2520Length%2520and%2520Line%2520Breaking%2Cless%2520is%2520not%2520always%2520possible.&usg=AOvVaw0vfbVgLUaRjaS-F-dB7X-M)

Reply:
Would you break the line with "\" or would you try to restructure it?

Reply:
definitly restructure, avoid line breaks (... \ ) whenever possible!
so i think just putting the raise warning on the next line would do it.